sensing/autoware_pointcloud_preprocessor/schema/ring_outlier_filter_node.schema.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Ring Outlier Filter Node",
+  "type": "object",
+  "definitions": {
+    "ring_outlier_filter": {
+      "type": "object",
+      "properties": {
+        "distance_ratio": {
+          "type": "number",
+          "description": "distance_ratio",
+          "default": "1.03",
+          "minimum": 0
+        },
+        "object_length_threshold": {
+          "type": "number",
+          "description": "object_length_threshold",
+          "default": "0.1",
+          "minimum": 0
+        },
+        "num_points_threshold": {
+          "type": "integer",
+          "description": "num_points_threshold",
+          "default": "4",
+          "minimum": 0
+        },
+        "max_rings_num": {
+          "type": "integer",
+          "description": "max_rings_num",
+          "default": "128",
+          "minimum": 1
+        },
+        "max_points_num_per_ring": {
+          "type": "integer",
+          "description": "Set this value large enough such that HFoV / resolution < max_points_num_per_ring",
+          "default": "4000",
+          "minimum": 0
+        },
+        "publish_outlier_pointcloud": {
+          "type": "boolean",
+          "description": "Flag to publish outlier pointcloud and visibility score. Due to performance concerns, please set to false during experiments.",
+          "default": "false"
+        },
+        "min_azimuth_deg": {
+          "type": "number",
+          "description": "The left limit of azimuth for visibility score calculation",
+          "default": "0.0",
+          "minimum": 0
+        },
+        "max_azimuth_deg": {
+          "type": "number",
+          "description": "The right limit of azimuth for visibility score calculation",
+          "default": "360.0",
+          "minimum": 0,
+          "maximum": 360
```

Comment:
I do not how schema handles this, but ROS2 is quite squeamish about how it parses ints and floats differently.

-> 360.0

Also, some numbers are between commas and other not. Which is the correct version? Make it consistent

Reply:
@knzo25 Thanks for the review. For the default value, from Amadeusz's suggestion, assigning the default values as strings will pass the number precisely. 

Minimum and maximum values do not accept `commas`. Also, from the website examples and by testing local, 360 and 360.0 are the same (as they are just checking for the boundary but not related to ros2 parsing?)

Reply:
I fixed the type to float in https://github.com/autowarefoundation/autoware.universe/pull/8468/commits/9b6614c2b00234daab3fba1af29883400854ab7b

Reply:
I do understand that it is not a problem in schema.
However, users would see the number and copy it (in some cases) resulting in it to fail in runtime